main/src/ca/uwaterloo/flix/language/errors/ResolutionError.scala

```diff
@@ -125,6 +125,44 @@ object ResolutionError {
     }
   }
 
+  /**
+    * Illegal Uninhabited Type Error.
+    * @param tpe the uninhabited type.
+    * @param loc the location where the error occurred.
+    */
+  case class IllegalUninhabitedType(tpe: Type, loc: SourceLocation) extends ResolutionError {
+    override def summary: String = "Illegal uninhabited type."
+    override def message: VirtualTerminal = {
+      val vt = new VirtualTerminal
+      vt << Line(kind, source.format) << NewLine
+      vt << ">> Illegal uninhabited type: '" << Red(tpe.toString) << "' with kind '" << Red(tpe.kind.toString) << "'." << NewLine
```

Comment:
We should use FormatType here instead of toString.

Maybe we also need a FormatKind?

Reply:
I think the kind formatting isn't complex enough to warrant a formatKind, though I think it would be nice to have a parallel structure and might be useful when/if kind vars become more widely used.

Reply:
OK. But you do want to use FormatType instead of toString right?

Reply:
Yep!